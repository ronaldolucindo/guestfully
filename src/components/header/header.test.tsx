import { render, screen } from '@testing-library/react'
import Header from './header'
import { BrowserRouter } from 'react-router-dom'
describe('Header', () => {
  it('renders logo with alt test', () => {
    render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    )
    expect(screen.getByTestId('Header.logo').getAttribute('alt')).toBeTruthy()
  })
  it('matches the logo', () => {
    render(
      <BrowserRouter>
        <Header />
      </BrowserRouter>,
    )

    expect(
      screen.getByTestId('Header.logo').getAttribute('alt'),
    ).toMatchSnapshot()
    expect(
      screen.getByTestId('Header.logo').getAttribute('src'),
    ).toMatchSnapshot()
  })
})
