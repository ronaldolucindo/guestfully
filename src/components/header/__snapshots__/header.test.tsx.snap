// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > matches the logo 1`] = `"Guestfully"`;

exports[`Header > matches the logo 2`] = `"/src/assets/logo.jpg"`;
